lib/manager/composer/extract.ts

```diff
@@ -81,7 +81,7 @@ function parseRepositories(
 export async function extractPackageFile(
   content: string,
   fileName: string
-): Promise<PackageFile | null> {
+): Promise<PackageFile<ComposerManagerData>> {
```

Comment:
```suggestion
): Promise<PackageFile<ComposerManagerData> | null> {
```
Can be `null` and is needed for more strict typecheck later